modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/ExecutionServiceImpl.java

```diff
@@ -496,23 +514,6 @@ private AsyncDataCursor<InternalSqlRow> executeKill(
         return new IteratorToDataCursorAdapter<>(ret, Runnable::run);
     }
 
-    private static RuntimeException convertDdlException(Throwable e) {
-        e = ExceptionUtils.unwrapCause(e);
-
-        if (e instanceof ConfigurationChangeException) {
-            assert e.getCause() != null;
-            // Cut off upper configuration error`s as uninformative.
-            e = e.getCause();
-        }
-
-        if (e instanceof IgniteInternalCheckedException) {
-            return new IgniteInternalException(INTERNAL_ERR, "Failed to execute DDL statement [stmt=" /* + qry.sql() */
-                    + ", err=" + e.getMessage() + ']', e);
-        }
-
-        return (e instanceof RuntimeException) ? (RuntimeException) e : new IgniteInternalException(INTERNAL_ERR, e);
```

Comment:
It looks like the same error handling should be removed when handling the KILL command :pensive: 
But we can probably do it separately later...

Reply:
I propose to do it separately in https://issues.apache.org/jira/browse/IGNITE-24387. It would be nice to revise test coverage for exception remapping.